ci/kokoro/macos/builds/bazel.sh

```diff
@@ -84,8 +83,7 @@ bazelisk build "${bazel_args[@]}" ...
 
 should_run_integration_tests() {
   if [[ -r "${GOOGLE_APPLICATION_CREDENTIALS}" && -r \
-    "${TEST_KEY_FILE_JSON}" && -r \
-    "${TEST_KEY_FILE_P12}" ]]; then
```

Comment:
fyi, the p12 key files do not exist anymore, so this test will be false and integration tests will not be run.

Reply:
Please ignore. I was confused.